fee_calculator/apps/calculator/fixtures/price.json

```diff
@@ -34062,7 +34062,7 @@
     "fee_type": 15,
     "unit": "DAY",
     "fixed_fee": "0.00000",
-    "fee_per_unit": "151.00000",
```

Comment:
👍 certainly looks like its in line with [The Criminal Legal Aid (Remuneration) Regulations 2013](http://www.legislation.gov.uk/uksi/2013/435/schedule/1/paragraph/21/made)